packages/module/src/ConsoleShared/src/components/layout/PageLayout.scss

```diff
@@ -1,29 +1,29 @@
 .pfext-page-layout {
   &__content {
-    padding: var(--pf-v5-global--spacer--lg);
+    padding: var(--pf-v6-global--spacer--lg);
     flex: 1;
   }
 
   &__header {
     padding-top: 1px;
-    background-color: var(--pf-v5-global--BackgroundColor--light-100);
+    background-color: var(--pf-v6-global--BackgroundColor--light-100);
```

Comment:
If this should just be the regular background color, for v6 it's `--pf-t--global--background--color--primary--default`